tests/nlp/checks/model_evaluation/weak_segment_performance_test.py

```diff
@@ -93,7 +93,7 @@ def test_multilabel_dataset(multilabel_mock_dataset_and_probabilities):
     # Assert
     assert_that(condition_result, has_items(
         equal_condition_result(is_pass=True,
-                               details='Found a segment with f1 macro score of 0.695 in comparison to an average '
+                               details='Found a segment with f1 macro score of 0.712 in comparison to an average '
```

Comment:
There is some instability in this test. I think perhaps we should change the test so not to specify the exact score...

Reply:
Okay, so how do we want to do it? Maybe jus checking the keys or something else?

Reply:
either that or matching the string without the number if possible

Reply:
Okay, so I have done regex-based matching so it will match the string with any number in place of 0.695.

Reply:
I think this can be merged now.